<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Asistencias</class>
 <widget class="QDialog" name="Asistencias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Asistencias</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="tx_buscar_asistencia">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_dni">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DNI</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_apellido">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apellido</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_nombre">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_entrada">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Entrada</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_salida">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Salida</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="tx_date">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QHeaderView::section {
    background-color:  rgb(7, 70, 124);
	color: rgb(255, 255, 255);
	font: 12pt &quot;Verdana&quot;;
    border: 0px solid #ff0000;
    height: 32px;
}

QTableWidget {
	font: 11pt &quot;Verdana&quot;;
    border: 1px solid;
}

QTableWidget::item{
    selection-background-color: rgb(7, 70, 124);
	selection-color: rgb(255, 255, 255);
}
</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="autoScroll">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideRight</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>12</number>
       </property>
       <property name="columnCount">
        <number>16</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>50</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>DNI</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Apellido</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nombre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Entrada</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Salida</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tardanza</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Restante</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>División</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Materia</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Estado</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fecha</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ciclo lectivo</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Sede</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Observación</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>id_clase</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="bt_reporte">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>39</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>410</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>18</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(7, 70, 124);
font: 18pt &quot;Verdana&quot;;
border-radius: 10px;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Generar reporte de asistencias</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
